lldb/source/Plugins/ExpressionParser/Clang/IRForTarget.cpp

```diff
@@ -514,7 +513,8 @@ bool IRForTarget::RewriteObjCConstString(llvm::GlobalVariable *ns_str,
            m_CFStringCreateWithBytes, CFSCWB_arguments,
            "CFStringCreateWithBytes",
            llvm::cast<Instruction>(
-               m_entry_instruction_finder.GetValue(function)));
+               m_entry_instruction_finder.GetValue(function))
+               ->getIterator());
```

Comment:
Similar story to above, `FindEntryInstruction` uses a debug-intrinsic skipping function, so there's no need to pass around the iterator as a position.